### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by r-ahmed2022**
In this pull request, I completed the following:-
1.  Created a basic page structure for the Leaderboard site.
2. Added basic CSS styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+understanding of Modules
+
+### Setup
+
+webpack setup
+
+### Install
+
+### Usage
+
```
Comment (TedLivist) [inline README.md]: - [ ] For subsequent projects that build on this one, the instruction for setting up and running the project would be the same so we can add them now. Kindly include detailed instructions on how to set up and run the app locally, i.e. from cloning to starting the app `(npm start)`

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
```
Comment (TedLivist) [inline README.md]: - [ ] To keep your documentation neat, I would recommend you remove generic messages from the template

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @r-ahmed2022,

Good job so far!
✔️ App is set up with webpack
✔️ UI matches wireframe
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (r-ahmed2022) [reply README.md]: Done as advised

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:
success
Hey @r-ahmed2022 :wave:

Great work on implementing the changes requested by previous reviewer. Your project is complete. There is nothing else to say about your project. :+1:
Go ahead and merge it. :muscle:

## Highlights :1st_place_medal:
:heavy_check_mark: Following GitFlow
:heavy_check_mark: Following Linters
:heavy_check_mark: Webpack Setup

### [Optional]

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**